specs/reverse_specs.rb

```diff
@@ -0,0 +1,28 @@
+require 'rspec'
+require_relative '../solver'
+
+describe Solver do
+  describe '#reverse' do
+    context 'when string is empty' do
+      it 'returns empty string' do
+        expect(Solver.reverse('')).to eq('')
+      end
+    end
+
+    context 'when string is not empty' do
+      it 'returns string in reverse' do
+        expect(Solver.reverse('hello')).to eq('olleh')
+      end
+    end
+
+    context 'when input is not a string' do
+      it 'should raise an error' do
+        expect { Solver.reverse(1) }.to raise_error(ArgumentError)
+      end
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/229286084-74ae6d12-ed79-4bc5-b76f-b5dbcd54a4d2.png)


- [ ] It would be better if these tests pass :+1: to make your app works as expected :+1: